src/components/userComponents/Exhibit/Exhibit.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Image from 'next/image';
+
+/**
+ *
+ * @param root0 passed in
+ * @param root0.title title of exhibit
+ * @param root0.description description of exhibit
+ * @param root0.category category of exhibit
+ * @param root0.image image
+ * @param root0.id id of exhibit
+ * @returns exhibit component
+ */
+export default function Exhibit({
+  title,
+  description,
+  image,
+  id,
+}: {
+  title: string;
+  description: string;
+  image: string;
+  id: string;
+}) {
+  return (
+    <ul key={id}>
+      <div className="w-[100%] px-4 pt-6 pb-7 bg-mint-cream rounded flex-col justify-start items-start gap-2.5 inline-flex mt-6">
+        <div className="flex-col justify-start items-start gap-5 flex">
+          <div className="justify-start items-center gap-2 inline-flex">
+            <div className="text-neutral-700 text-lg font-bold font-['Lato']">
```

Comment:
text styling should likely be done with <h_> tags (in this case h2), since it is based off the H2 text style in the design. i haven't looked through the way styling is done in the app though, so this might be wrong.